### PR TITLE
docs(how-to guides): removed 'how-to' prefix from subtitles

### PR DESCRIPTION
**Pull request by kminoda**
Signed-off-by: kminoda <koji.m.minoda@gmail.com>

## Description
Removed "how to ..." prefix from how-to guides subsections. 

Reason
- There were several types of formats in the subtitles ("How to [verb]..." & "[verb] ...") and this should be unified into one format.
- Adding "How to..." to all the subtitles is redundant.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by LalithVipulananthan** [comment]
@kminoda If we are trying to follow the naming convention used by the ROS 2 tutorials, then we need to change the titles further. For example, `Integrate Autoware with your vehicle` should be `Integrating Autoware with your vehicle` and so on.

**Comment by kminoda** [comment]
@LalithVipulananthan You're right, thanks. I updated the subtitles.

File: docs/how-to-guides/calibrating-your-sensors.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# How to calibrate your sensors
+# Calibrating your sensors
```
Comment (LalithVipulananthan) [inline docs/how-to-guides/calibrating-your-sensors.md:1]: I think that `Calibrating sensors` is fine. 

```suggestion
# Calibrating sensors
```

**Review by LalithVipulananthan — APPROVED** [review]
LGTM!

**Review by kenji-miyake — APPROVED** [review]

**Pull request merged**